### PR TITLE
Update readme

### PR DESCRIPTION
**Pull request by dkozickis**
Update README.md with usage examples.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,20 +1,65 @@
...
 yarn add @date-fns/upgrade
 ```
 
+## Usage
+
+### convertTokens
+
+`convertTokens` is a helper function used for 2nd argument of `format` function to convert date tokens like `YYYY` to new format. See [this post](https://blog.date-fns.org/post/unicode-tokens-in-date-fns-v2-sreatyki91jg) for more details.
+
+Obviously, you can convert tokens manually :)
```
Comment (kossnocorp) [inline README.md]: I think it's redundant ;-)

File: README.md [hunk 2]
```diff
@@ -1,20 +1,65 @@
...
+- 'YYYY',
++ convertTokens('YYYY'),
+)
+```
+
+### legacyParse
+
+`date-fns@2.x` functions don't accept string as arguments any more (see [CHANGELOG](https://github.com/date-fns/date-fns/blob/master/CHANGELOG.md#changed)), `legacyParse` is used to simplify that transition. See [this post](https://blog.date-fns.org/post/we-cut-date-fns-v2-minimal-build-size-down-to-300-bytes-and-now-its-the-smallest-date-library-18f2nvh2z0yal) for details on "why" this was done.
+
+```diff
```
Comment (kossnocorp) [inline README.md:39]: Please add that `legacyParse` uses the v1 algorithm to convert arguments to dates.

File: README.md [hunk 3]
```diff
@@ -1,20 +1,65 @@
...
+const formattedDate = format(
+- '2014',
++ legacyParse('2014'),
+  'YYYY',
+)
+```
+
+### legacyParseMap
+
+`legacyParseMap` is used same as `legacyParse` but for arguments that accept arrays of data
```
Comment (kossnocorp) [inline README.md]: Did you mean arrays of dates? Otherwise, you can simply say "that accept arrays."

**Review by kossnocorp — APPROVED** [review]
Thank you for that! Please fix my comments and it's ready to go!

File: README.md (hunk 1 above)
Comment (dkozickis) [reply README.md]: 🤷‍♂

File: README.md (hunk 2 above)
Comment (dkozickis) [reply README.md:39]: Mentioned.

File: README.md (hunk 3 above)
Comment (dkozickis) [reply README.md]: I did... left just the "that accepts arrays".

**Pull request merged**